app/src/main/java/com/quizarena/sessions/creation/CreateSessionApi.kt

```diff
@@ -0,0 +1,21 @@
+package com.quizarena.sessions.creation
+
+class CreateSessionApi {
```

Comment:
Das würde ich einfach mit in die SessionApi schreiben

Reply:
Hab ich verschoben